diff --git a/partial-modern/PartialModern.csproj b/partial-modern/PartialModern.csproj
--- a/partial-modern/PartialModern.csproj
+++ b/partial-modern/PartialModern.csproj
@@ -2,5 +2,7 @@
   <PropertyGroup>
     <TargetFramework>net8.0</TargetFramework>
     <OutputType>Library</OutputType>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
   </PropertyGroup>
 </Project>
