diff --git a/fsharp/BrokenOrder/BrokenOrder.fsproj b/fsharp/BrokenOrder/BrokenOrder.fsproj
--- a/fsharp/BrokenOrder/BrokenOrder.fsproj
+++ b/fsharp/BrokenOrder/BrokenOrder.fsproj
@@ -4,8 +4,8 @@
     <TargetFramework>net8.0</TargetFramework>
   </PropertyGroup>
   <ItemGroup>
+    <Compile Include="Domain.fs" />
     <Compile Include="Services.fs" />
-    <Compile Include="Domain.fs" />
     <Compile Include="Program.fs" />
   </ItemGroup>
 </Project>
